integration-tests/spring-data-jpa/src/test/java/io/quarkus/it/spring/data/jpa/CountryResourceTest.java

```diff
@@ -79,8 +79,9 @@ void testGetOne() {
                 .statusCode(200)
                 .body(containsString("Greece"));
 
-        when().get("/country/getOne/100").then()
-                .statusCode(500);
+        //TODO re-enable once HHH-16669
+        //  when().get("/country/getOne/100").then()
+        //  .statusCode(500);
```

Comment:
https://hibernate.atlassian.net/browse/HHH-16669 was supposedly fixed in ORM 6.2.4, so I think this should be uncommented?

Reply:
Right - removed! sorry